x-pack/plugins/ml/public/application/util/recently_accessed.ts

```diff
@@ -37,7 +37,7 @@ export function addItemToRecentlyAccessed(page: string, itemId: string, url: str
       return;
   }
 
-  url = `ml#/${page}/${url}`;
+  url = `${page}/${url}`;
```

Comment:
This fails if clicking this link from outside the ML app. E.g. go back to the Kibana home page, and click on an ML recently accessed link, the `ml` is now needed in the url:

![image](https://user-images.githubusercontent.com/7405507/93229252-4d166d00-f76e-11ea-977a-bf035b80a557.png)

Reply:
Thanks for catching that. I didn't test this outside of the ML plugin so the path was different. Fixed the issue here https://github.com/elastic/kibana/pull/76735/commits/ba56a81be63cd5517d3dd4be1fc9d2409b26b351